aquacrop/initialize/read_model_parameters.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 import pandas as pd
+from dateutil.relativedelta import relativedelta
```

Comment:
This will no longer be necessary if start_end_years is an array. See comment bellow.